src/app/app.component.html

```diff
@@ -126,7 +126,13 @@
 
 <router-outlet></router-outlet>
 
-<biggive-footer heading-level=3>
+<biggive-footer
+  heading-level=3
+  primary-nav-title="Match Funding Opportunities"
+  secondary-nav-title="Resources"
+  tertiary-nav-title="About"
+  postscript-nav-title="postscript"
```

Comment:
Not sure about this label. Should we request copy that gives a better idea what's inside the area?

Reply:
Yes let me ask on Slack.